base/multidimensional.jl

```diff
@@ -940,3 +940,178 @@ If `dim` is specified, returns unique regions of the array `itr` along `dim`.
         @nref $N A d->d == dim ? sort!(uniquerows) : (1:size(A, d))
     end
 end
+
+"""
+    groupslices(A, dim)
```

Comment:
In which *.rst file should the signature line for this function live?  At present, there is no multidimensional.rst file.